doc/flame/examples/lib/drag_events.dart

```diff
@@ -127,7 +127,7 @@ class Trail extends Component {
       final path = _paths[i];
       final opacity = _opacities[i];
       if (opacity > 0) {
-        _linePaint.color = _color.withOpacity(opacity);
+        _linePaint.color = _color.withValues(alpha: opacity);
```

Comment:
I might be wrong, but iirc, alpha and opacity are not 1:1 changeable are they?

Reply:
Oh, nvm: https://docs.flutter.dev/release/breaking-changes/wide-gamut-framework#withopacity-migration